app/models/pointless_feedback/message.rb

```diff
@@ -13,9 +16,11 @@ class Message < ActiveRecord::Base
     private
 
     def export_feedback
-      if PointlessFeedback.email_feedback
-        FeedbackMailer.feedback(self).deliver
-      end
+      return unless PointlessFeedback.email_feedback
+
+      # Support Rails < 4.2 and >= 4.2 delivery options
+      mailer = FeedbackMailer.feedback(self)
+      mailer.respond_to?(:deliver_now) ? mailer.deliver_now : mailer.deliver
```

Comment:
Yea, `deliver_now` and `deliver_later`.  Sucks the API doesn't support `deliver` as an alias to `deliver_now`.